fluxacademy.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<head>
+<title>Flux Academy</title>
+    <style>
+        body {
+            margin: 0;
+        }
+        .container {
+         padding: 1%;
+            box-sizing:border-box;
+            
+    }
+        .header {
+            height: 100px;
+            text-align:center;
+    }
+
+        .left-col, .right-col {
+        width: 25%;
+        float: left;
+    }
+        .center-col {
+            width:50%;
+            float:left;
+        }
+
```

Comment:
For all of your css, the indentation is off for nearly every selector.

Should look like this:

```
     .someClass {
         ... your css here
     }
```

... and NOT:

```
      .someClass {
               .... your css
}
```

Reply:
There is only one space, tested by moving property back one space then it was inline with class(not nested).

Reply:
Follow the exact indentation that's happening between lines 5-7 for every selector.
You can see in nearly every selector, something is off (whether it's too much indent or lack of it).
Build the habit of visually aligning things like this when you look over your work again.